birdclef/workflows/templates/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
```

Comment:
You may want to rename this `index.html.j2` to avoid pre-commit from breaking the templating stuff.